docs/azure/sdk/snippets/protocol-convenience-methods/AzureCoreProtocol/Program.cs

```diff
@@ -15,19 +16,24 @@
 });
 
 // Call the protocol method
-Response response = safetyClient.AnalyzeText(
-    prompt, 
-    new RequestContext()
+Response response = client.AnalyzeText(
+    prompt,
+    new RequestContext
     {
-        ErrorOptions = ErrorOptions.NoThrow
+        ErrorOptions = ErrorOptions.NoThrow,
     });
 
 // Any non-200 response code from Azure AI Content Safety AnalyzeText REST API isn't considered a success response.
 // See REST API details at https://azure-ai-content-safety-api-docs.developer.azure-api.net/api-details#api=content-safety-service-2023-10-01&operation=TextOperations_AnalyzeText
-if (response.Status != 200) 
+if (response.Status != 200)
 {
-    throw new RequestFailedException(response); 
+    throw new RequestFailedException(response);
 }
 
+dynamic content = response.Content.ToDynamicFromJson(JsonPropertyNames.CamelCase);
```

Comment:
It might be helpful to add a comment here with a short explanation that the `dynamic` instance returned from `ToDynamicFromJson` lets you use response content like a convenience model, which helps you evolve code from protocol to convenience with minimal code changes as convenience layers are added to libraries.